bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/handler/StatusHandler.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.easee.internal.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.easee.internal.connector.WebInterface;
```

Comment:
Compiler warning here since this import is unused.

Reply:
it is mentioned in the javadoc and thus imported by Eclipse. It will be readded on save when I try to remove it.

Reply:
I changed the javadoc comment and removed the import.